### PR TITLE
docs: added Renovate Version info to Mend-hosted Apps config page

### PR DESCRIPTION
**Pull request by justo-mend**
## Changes

Added information about the Renovate Version of the Mend-hosted apps to the Mend-hosted Apps Configuration doc.
Linked similar section of FAQs to this new docs section.

## Context

Users have expressed interest in knowing about the Renovate version in the Mend-hosted apps.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/mend-hosted/hosted-apps-config.md [hunk 1]
```diff
@@ -17,6 +17,36 @@ The Renovate logs for the Mend-hosted apps are on the [Mend Developer Portal](ht
 
 Reading the logs can help you understand the configuration that Renovate used.
 
+## Renovate Version
+
+The Renovate version used by the Mend-hosted apps is updated manually by the maintainers of the app.
+The maintainers don't follow any release schedule or release cadence, but try to update at least once a week.
+This means the Mend Renovate App can lag a few hours to a week behind the open source version.
```
Comment (rarkins) [inline docs/usage/mend-hosted/hosted-apps-config.md:24]: This still duplicates the FAQ content too much

Comment (justo-mend) [reply docs/usage/mend-hosted/hosted-apps-config.md:24]: Would you rather the content be mainly in the FAQs or in the Mend-hosted apps section?

Comment (rarkins) [reply docs/usage/mend-hosted/hosted-apps-config.md:24]: Mend-hosted. FAQ is a catchall anyway

**Review by rarkins — CHANGES_REQUESTED** [review]
Deduplicate/remove FAQ questions relating to Mend app

File: docs/usage/mend-hosted/hosted-apps-config.md (hunk 1 above)
Comment (justo-mend) [reply docs/usage/mend-hosted/hosted-apps-config.md:24]: Removed duplication in FAQs. Replaced FAQ content with a link to the Mend-hosted App Config.

File: docs/usage/faq.md
Comment (rarkins) [inline docs/usage/faq.md:1]: Please remove these FAQ questions/headings altogether

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/usage/faq.md
Comment (justo-mend) [reply docs/usage/faq.md:1]: Done. Removed questions from the FAQs.

**Review by rarkins — APPROVED** [review]

**Pull request merged**